codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -1178,14 +1174,14 @@ public void process() {
             }
         }
 
-        public void deactivate(final DefaultStream stream) {
+        public void deactivate(final DefaultStream stream, final Iterator<?> itr) {
```

Comment:
Can you remind me why this method is public?  I don't think we should be exposing these iterators on public interfaces.